packages/config/config/devices/0x0258/nas-pd07u1.json

```diff
@@ -0,0 +1,213 @@
+	"description": "5 in 1 PIR Motion Sensor",
+	"devices": [
+		{
+			"productType": "0x0020",
+			"productId": "0x0718"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
+		},
+		{
+			"#": "2",
+			"label": "Motion Detection",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"$import": "templates/shenzhen_neo_template.json#motion_prevent_retrigger"
+		},
+		{
+			"#": "4",
+			"label": "Require Luminance Change for Basic Sets on Motion",
+			"description": "If enabled, a Basic Set will be set if the ambient light is below the level defined in Parameter 16",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Binary Sensor Report on Motion",
+			"description": "If enabled, a Binary Sensor report will be sent on Motion",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+        {
+			"#": "6",
+			"label": "Motion Sensor Sensitivity",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 2,
+			"unsigned": true
+		},
+		{
+			"#": "7",
+			"label": "Temperature Offset Value",
+			"valueSize": 1,
+			"unit": "0.1 °C/F",
+			"minValue": -120,
+			"maxValue": 120,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "8",
+			"label": "Humidity Offset Value",
+			"valueSize": 1,
+			"unit": "0.1 %",
+			"minValue": -120,
+			"maxValue": 120,
+			"defaultValue": 0,
+			"unsigned": false
+		},
+		{
+			"#": "9",
+			"label": "Temperature Change Report Threshold",
+			"valueSize": 1,
+			"unit": "0.1 °C/F",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10
+		},
+		{
+			"#": "10",
+			"label": "Humidity Change Report Threshold",
+			"valueSize": 1,
+			"unit": "0.1 %",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 20
+		},
+		{
+			"#": "11",
+			"label": "Luminance Change Report Threshold",
+			"valueSize": 1,
+			"unit": "lux",
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 50
+		},
+		{
+			"#": "12",
+			"label": "Association Group 2: Basic Set Level",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 100
+				}
```

Comment:
```suggestion
				{
					"label": "On (255)",
					"value": 100
				}
```

Reply:
The values were referring to the command that gets sent, not turning the feature on/off.